db.py

```diff
@@ -1,2 +1,19 @@
 from flask_sqlalchemy import SQLAlchemy
+# from models.user import UserModel
+# from models.property import PropertyModel
+# from models.revoked_tokens import RevokedTokensModel
 db = SQLAlchemy()
+
+# def seedData():
```

Comment:
Should we just delete all these commented lines? Or replace these with the ones added in app.py?

Reply:
yeah, db.py should just be the two lines of code